src/App.tsx

```diff
@@ -18,15 +20,11 @@ const router = createBrowserRouter([
       { path: "/blogs", element: <Blogs /> },
       { path: "/blogs/:id", element: <BlogPage /> },
       { path: "/about", element: <About /> },
+      { path: "/about", element: <AboutUs /> },
       { path: "/albums", element: <Albums /> },
       { path: "/events", element: <Events /> },
       { path: "/donate", element: <Donate /> },
-      {
-        path: "/contact",
-        element: (
-         <Contact />
-        ),
-      }
+      {path: "/contact",element: (<Contact />),}
```

Comment:
Use prettier or the tool that has been used in the current repo, let's follow the formatting standards. @mr0bn0xi0us 

For details regarding formatting, you can talk with @ankurgajurel